### DISCUSSION TITLE
N00b from scratch

### DISCUSSION THREAD

**Discussion by eddow** [orta/vscode-jest#962] [Q&A] [answered]
Hi here,

I feel like a newbie as I used jest before but am completely unable to start using it from scratch. I spent two hours reading boilerplates, following all indications, .... but `describe` won't be recognized. (nor `test` nor `expect`)

I made a really minimalist ts-jest example [here](https://github.com/eddow/eddow-bug-expo) if someone could help me get this thorn out of my foot ...

**Comment by connectdotz** [comment, marked as answer]
looks like you have a typo in `tsconfig.json`:
```
"typeRoots": ["../node_modules/@types", "."]
```
I think you mean:
```
"typeRoots": ["./node_modules/@types", "."]
```
